src/cider/nrepl/middleware/out.clj

```diff
@@ -18,12 +18,14 @@
 
 (declare unsubscribe-session)
 
+(defonce org-out {:out *out*
+                  :err *err*})
+
 (defn original-output
   "Store the values of the original output streams so we can refer to them."
-  ^PrintWriter
+  ^Writer
```

Comment:
Let's remove the type hint entirely to make it just like the previous version.

Reply:
I think if don't use type hint, no reason to wrap a map with the function.

Reply:
Ok, let's prefer `defonce` to `def` and use the following docstring:

```
Store the values of the original output streams so we can refer to them.
Please make sure they are not recomputed at runtime.
```

Reply:
(The idea is that we make it as hard as possible to make the same mistake again)